src/components/AboutUsIntroduction/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const AboutUsIntroduction = () => {
+  const { t } = useTranslation('about-us')
+  return (
```

Comment:
Please add an empty space before returning **JSX** element 

```suggestion

  return (
```